src/plan/mutator_context.rs

```diff
@@ -278,7 +314,24 @@ pub trait MutatorContext<VM: VMBinding>: Send + 'static {
         offset: usize,
         allocator: AllocationSemantics,
     ) -> Address;
-    /// The slow path allocation. This is only useful when the binding
+    /// Allocate memory for an object. This is NOT a GC safepoint. If the allocation fails,
```

Comment:
We don't want to use the safepoint terminology here like we discussed in the meeting, right? Same for the functions below.